sync/config.py

```diff
@@ -39,25 +39,12 @@ def customise_sources(cls, init_settings, env_settings, file_secret_settings):
 
 
 class Configuration(BaseSettings):
-    default_project_url: Union[str, None] = Field(
-        description="default location for Sync project data"
-    )
```

Comment:
Removing this configuration option, since it doesn't seem to really be used anywhere anyway (and because the ticket is asking to remove it)